src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -516,9 +516,13 @@ export default function V2ProjectReconfigureModal({
       onOk={reconfigureFundingCycle}
       onCancel={handleGlobalModalClose}
       okText={t`Deploy new project configuration`}
-      okButtonProps={{ disabled: !fundingHasSavedChanges }}
+      okButtonProps={{
+        disabled: !fundingHasSavedChanges,
+        style: { marginBottom: '15px' },
+      }}
       confirmLoading={reconfigureTxLoading}
       width={650}
+      style={{ paddingBottom: 20 }}
```

Comment:
**question** could we apply paddingBotton of `35px` here and get the same outcome, without modifying the button style?

Reply:
They are different, the padding creates space between and modal and the browser window, margin creates space between the OK button and the edge of modal. I've added a paddingTop for consistency